libs/pubsub/query/query_test.go

```diff
@@ -2,15 +2,38 @@ package query_test
 
 import (
 	"fmt"
+	"strings"
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-
+	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/libs/pubsub/query"
 )
 
+func expandEvents(flattenedEvents map[string][]string) []abci.Event {
```

Comment:
I added this helper method to avoid having to modify the massive `TestMatches` which would be a huge PIA.